writing-specifications.rst

```diff
@@ -48,31 +63,51 @@ The following is an example of a Gauge specification:
 
     The admin user must be able to search for available products on the search page
 
-    * User must be logged in as "admin"
-    * Open the product search page
-
     ## Successful search
     Tags: successful
 
     For an existing product name, the search result will contain the product name
 
+    * User must be logged in as "admin"
+    * Open the product search page
     * Search for product "Die Hard"
     * "Die Hard" should show up in the search results
 
+    
     ## Unsuccessfull search
 
     On an unknown product name search the search results will be empty
 
+    * User must be logged in as "admin"
+    * Open the product search page
     * Search for product "unknown"
     * The search results will be empty
 
+For more information about Markdown, see `Markdown Syntax <https://daringfireball.net/projects/markdown/syntax>`__.
 
-Specification Heading
-^^^^^^^^^^^^^^^^^^^^^
+Components of a specification
+-----------------------------
+A specification comprises of different sections; some of which are mandatory and few are optional.
```

Comment:
`A specification comprises of different sections` might not be clear to all users. Can we replace `comprises` with a simpler word?

Reply:
How about:
A specification consists of different sections
OR
A specification contains different sections

Reply:
`A specification consists of different sections` works fine